clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/arrays/ArraysExtensions.scala

```diff
@@ -0,0 +1,41 @@
+package org.broadinstitute.clio.transfer.model.arrays
+
+/**
+  *  Extensions for files in the arrays index.
+  */
+object ArraysExtensions {
+
+  val BpmExtension = ".bpm"
+
+  val CsvExtension = ".csv"
+
+  val DictExtension = ".dict"
```

Comment:
Many of these extensions are not arrays specific.

Reply:
I would just make this `FileExtension` and not try to organize extensions by index type. Using singular rather than plural and dropping the redundant `Extension` from the name would make them easier to read, IMO, e.g. `FileExtension.Bpm` instead of `FileExtensions.BpmExtension`.